### PR TITLE
[GEOS-8571] Autocomplete support for dropdowns with long lists of values

### PR DESCRIPTION
**Pull request by aaime**

### CODE REVIEW

File: src/web/core/src/test/java/org/geoserver/web/GeoServerWicketTestSupport.java [hunk 1]
```diff
@@ -118,7 +123,7 @@ public Component findComponentByContent(MarkupContainer root, Object content, Cl
         root.visitChildren(componentClass, finder);
         return finder.candidate;
     }
-    
+
```
Comment (tbarsballe) [inline src/web/core/src/test/java/org/geoserver/web/GeoServerWicketTestSupport.java]: This file seems to contain only non-functional changes

Comment (aaime) [reply src/web/core/src/test/java/org/geoserver/web/GeoServerWicketTestSupport.java]: Yep, will remove it from changeset

**Comment by aaime** [comment]
Ah yep, the layer dropdowns for vector/raster layers are indeed a good candidate for switch, I'll look into that.

**Comment by aaime** [comment]
Added support for WPS builder usage, minor CSS tweaks (in WPS there is a native and a select2 side by side, they were misaligned), and... merged!

**Pull request merged**